pkg/ebpf/tracee.go

```diff
@@ -59,6 +60,45 @@ type fileExecInfo struct {
 	Hash      string
 }
 
+// fileUniqueId is an unique identifier for a file, based on its device and inode.
+type fileUniqueId struct {
```

Comment:
I have a problem with calling this `FileUniqueId` as this is incorrect.
The hash is also dependent on the `ctime`, so it should be included.
We have the concept of file unique ID in the eBPF code, you can take a look over there - https://github.com/aquasecurity/tracee/blob/main/pkg/ebpf/c/types.h#L216

Reply:
Good point. I can change it to embed ctime, and instead of dev/inode have a string with the parsed origin as mentioned in https://github.com/aquasecurity/tracee/pull/3746#discussion_r1418838352